### ISSUE TITLE
Overlay and Keystrokes

### ISSUE THREAD

**Issue by xbc1** [SnosMe/awakened-poe-trade#788]
Hi,
OS: Ubuntu 22.04.1 LTS.
I am running PoE in Windowed Fullscreen, it is the same for Windowd, in FUllscreen it doesn't work at all but that doesn't matter.
The Overlay works in a strange way:
1. CTRL + D = Overlay works fine BUT after you move the mouse and it goes away PoE won't get any key strokes anymore, I can fix that by clicking anywhere on the second screen OR 
2. SHIFT + SPACE = no settings overlay also no supporters visible, mouse changes to OS mouse instead of PoE Mouse.
After that click anywhere in PoE so that the non visible overlay goes away and I get back to step 1, also after that SHIFT + SPACE opens the settings overlay just fine.

CTRL + ALT + D works just fine all the time, if you didn't do step 1 in the first place.
Any Ideas?

**Comment by justin8** [comment]
I see the same behaviour; pressing shift+space three times seems to make the overlay work as expected.

However I also found that it was absorbing all keypresses; space/esc/enter/etc would all not work any more in PoE

**Comment by xbc1** [comment]
> 
Doesthis behaviour get triggered when you switch zones ingame?
Because it does for me.

**Comment by justin8** [comment]
No, not that I’ve seen. What I’ve seen is it happens inconsistently only when I alt+tab or click a window on another monitor. But not always. Sometimes I can go a few hours without it breaking, other times it captures all input 10 times in a row.

**Comment by Myricaulus** [comment]
Does this tool use x11 automation? Since the new ubuntu is wayland based there are now several security "features" preventing some automation.

**Comment by SnosMe** [comment]
@Myricaulus works only with x11, and even then it can be luck based

**Comment by phobos-nik** [comment]
I have the same trouble using appimage with Fedora 37 (Wayland with NVidia propriental drivers).

**Comment by DeftlyMoving** [comment]
Reporting the same issue here, pressing shift + space three times resolves it for me.

OS: Pop OS 22.04 LTS
